test/abi/macOS/x86_64/stdlib.swift

```diff
@@ -562,6 +562,7 @@ Added: __swift_exceptionPersonality
 Added: _swift_willThrowTypedImpl
 Added: __swift_willThrowTypedImpl
 Added: __swift_enableSwizzlingOfAllocationAndRefCountingFunctions_forInstrumentsOnly
+Added: _swift_clearSensitive
 
```

Comment:
@eeckstein I think this change was left out in https://github.com/apple/swift/commit/ce33d47a4c3994052e57f8d6eaccda2700225c77. Added here.